<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MIDI to OBS Configurator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Action</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Combo_InputDevices">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>280</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Combo_OutputDevices">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>330</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>310</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Device</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Actions</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>260</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Device</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_deleteAction">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Action</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>511</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QPushButton" name="btn_RecordInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Record Input</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="DevID">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="addr">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="InputTypeSelector">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Button</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fader</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Device</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>type</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineType">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="Combo_Action">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Dnote">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Note</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="list_action">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>761</width>
      <height>192</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>10</horstretch>
      <verstretch>10</verstretch>
     </sizepolicy>
    </property>
    <column>
     <property name="text">
      <string>msg_type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>msgNoC</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Input Type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Action</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Device ID</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMIDI_to_OBS_Configurator">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuMIDI_to_OBS_Configurator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>InputTypeSelector</sender>
   <signal>activated(int)</signal>
   <receiver>Combo_Action</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_RecordInput</sender>
   <signal>toggled(bool)</signal>
   <receiver>btn_RecordInput</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
